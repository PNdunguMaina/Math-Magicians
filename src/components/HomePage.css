.header {
  font-family: Roboto, sans-serif;
  color: red;
  font-weight: 800;
}

.body {
  color: black;
  padding-top: 20px;
}

@media (max-width: 400px) {
  .header {
    font-size: x-large;
    font-weight: 700;
  }
}

@media (min-width: 975px) {
  .body {
    font-size: 2rem;
    letter-spacing: 2px;
  }
}
