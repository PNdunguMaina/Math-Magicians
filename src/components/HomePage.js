import React from 'react';
import styles from './HomePage.module.css';

function HomePage() {
  return (
    <div className={styles.homePageContainer}>
      <header className={styles.homePageHeader}>
        <h1>Welcome to our page</h1>
      </header>
      <div className={styles.homePageContent}>
        <p>
          Lorem ipsum dolor sit amet, consectetur adipiscing elit. Sed dapibus,
          nisi nec elementum fermentum, quam nisl sagittis felis, a tempor
          libero risus id nibh. Mauris lacus tellus, tristique non pharetra sit
          amet, placerat interdum lorem. Donec vitae quam ex. Etiam ultricies,
          leo sit amet sagittis lobortis, velit quam sodales nisi, vel cursus
          augue nibh vitae nisi. Vestibulum ante ipsum primis in faucibus orci
          luctus et ultrices posuere cubilia curae; Cras pellentesque ligula a
          metus porttitor posuere. Aenean in aliquet turpis. <br />
          Lorem ipsum dolor sit amet, consectetur adipiscing elit. Sed dapibus,
          nisi nec elementum fermentum, quam nisl sagittis felis, a tempor
          libero risus id nibh. Mauris lacus tellus, tristique non pharetra sit
          amet, placerat interdum lorem. Donec vitae quam ex. Etiam ultricies,
          leo sit amet sagittis lobortis, velit quam sodales nisi, vel cursus
          augue nibh vitae nisi. Vestibulum ante ipsum primis in faucibus orci
          luctus et ultrices posuere cubilia curae; Cras pellentesque ligula a
          metus porttitor posuere. Aenean in aliquet turpis.
        </p>
      </div>
    </div>
  );
}

export default HomePage;
