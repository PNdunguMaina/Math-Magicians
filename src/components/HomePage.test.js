/* eslint-disable no-undef */
import React from 'react';
import { render } from '@testing-library/react';
import { BrowserRouter as Router } from 'react-router-dom';

import HomePage from './HomePage';

it('renders HomePage as per expectation', () => {
  const tree = render(
    <Router>
      <HomePage />
    </Router>,
  );
  expect(tree).toMatchSnapshot();
});
