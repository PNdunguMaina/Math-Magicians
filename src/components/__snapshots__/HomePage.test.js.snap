// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders HomePage as per expectation 1`] = `
Object {
  "asFragment": [Function],
  "baseElement": <body>
    <div>
      <div
        class="homePageContainer"
      >
        <header
          class="homePageHeader"
        >
          <h1>
            Welcome to our page
          </h1>
        </header>
        <div
          class="homePageContent"
        >
          <p>
            Lorem ipsum dolor sit amet, consectetur adipiscing elit. Sed dapibus, nisi nec elementum fermentum, quam nisl sagittis felis, a tempor libero risus id nibh. Mauris lacus tellus, tristique non pharetra sit amet, placerat interdum lorem. Donec vitae quam ex. Etiam ultricies, leo sit amet sagittis lobortis, velit quam sodales nisi, vel cursus augue nibh vitae nisi. Vestibulum ante ipsum primis in faucibus orci luctus et ultrices posuere cubilia curae; Cras pellentesque ligula a metus porttitor posuere. Aenean in aliquet turpis.
            <br />
            Lorem ipsum dolor sit amet, consectetur adipiscing elit. Sed dapibus, nisi nec elementum fermentum, quam nisl sagittis felis, a tempor libero risus id nibh. Mauris lacus tellus, tristique non pharetra sit amet, placerat interdum lorem. Donec vitae quam ex. Etiam ultricies, leo sit amet sagittis lobortis, velit quam sodales nisi, vel cursus augue nibh vitae nisi. Vestibulum ante ipsum primis in faucibus orci luctus et ultrices posuere cubilia curae; Cras pellentesque ligula a metus porttitor posuere. Aenean in aliquet turpis.
          </p>
        </div>
      </div>
    </div>
  </body>,
  "container": <div>
    <div
      class="homePageContainer"
    >
      <header
        class="homePageHeader"
      >
        <h1>
          Welcome to our page
        </h1>
      </header>
      <div
        class="homePageContent"
      >
        <p>
          Lorem ipsum dolor sit amet, consectetur adipiscing elit. Sed dapibus, nisi nec elementum fermentum, quam nisl sagittis felis, a tempor libero risus id nibh. Mauris lacus tellus, tristique non pharetra sit amet, placerat interdum lorem. Donec vitae quam ex. Etiam ultricies, leo sit amet sagittis lobortis, velit quam sodales nisi, vel cursus augue nibh vitae nisi. Vestibulum ante ipsum primis in faucibus orci luctus et ultrices posuere cubilia curae; Cras pellentesque ligula a metus porttitor posuere. Aenean in aliquet turpis.
          <br />
          Lorem ipsum dolor sit amet, consectetur adipiscing elit. Sed dapibus, nisi nec elementum fermentum, quam nisl sagittis felis, a tempor libero risus id nibh. Mauris lacus tellus, tristique non pharetra sit amet, placerat interdum lorem. Donec vitae quam ex. Etiam ultricies, leo sit amet sagittis lobortis, velit quam sodales nisi, vel cursus augue nibh vitae nisi. Vestibulum ante ipsum primis in faucibus orci luctus et ultrices posuere cubilia curae; Cras pellentesque ligula a metus porttitor posuere. Aenean in aliquet turpis.
        </p>
      </div>
    </div>
  </div>,
  "debug": [Function],
  "findAllByAltText": [Function],
  "findAllByDisplayValue": [Function],
  "findAllByLabelText": [Function],
  "findAllByPlaceholderText": [Function],
  "findAllByRole": [Function],
  "findAllByTestId": [Function],
  "findAllByText": [Function],
  "findAllByTitle": [Function],
  "findByAltText": [Function],
  "findByDisplayValue": [Function],
  "findByLabelText": [Function],
  "findByPlaceholderText": [Function],
  "findByRole": [Function],
  "findByTestId": [Function],
  "findByText": [Function],
  "findByTitle": [Function],
  "getAllByAltText": [Function],
  "getAllByDisplayValue": [Function],
  "getAllByLabelText": [Function],
  "getAllByPlaceholderText": [Function],
  "getAllByRole": [Function],
  "getAllByTestId": [Function],
  "getAllByText": [Function],
  "getAllByTitle": [Function],
  "getByAltText": [Function],
  "getByDisplayValue": [Function],
  "getByLabelText": [Function],
  "getByPlaceholderText": [Function],
  "getByRole": [Function],
  "getByTestId": [Function],
  "getByText": [Function],
  "getByTitle": [Function],
  "queryAllByAltText": [Function],
  "queryAllByDisplayValue": [Function],
  "queryAllByLabelText": [Function],
  "queryAllByPlaceholderText": [Function],
  "queryAllByRole": [Function],
  "queryAllByTestId": [Function],
  "queryAllByText": [Function],
  "queryAllByTitle": [Function],
  "queryByAltText": [Function],
  "queryByDisplayValue": [Function],
  "queryByLabelText": [Function],
  "queryByPlaceholderText": [Function],
  "queryByRole": [Function],
  "queryByTestId": [Function],
  "queryByText": [Function],
  "queryByTitle": [Function],
  "rerender": [Function],
  "unmount": [Function],
}
`;
